core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-state-controller-seo/dot-edit-page-state-controller-seo.component.spec.ts

```diff
@@ -158,7 +171,8 @@ describe('DotEditPageStateControllerSeoComponent', () => {
                 expect(dotTabButtons).toBeDefined();
                 expect(dotTabButtons.options).toEqual([
                     { label: 'Edit', value: 'EDIT_MODE', disabled: false },
-                    { label: 'Preview', value: 'PREVIEW_MODE', disabled: false }
+                    { label: 'Preview', value: 'PREVIEW_MODE', disabled: false },
+                    { label: 'Live', value: 'ADMIN_MODE', disabled: false }
```

Comment:
@hmoreras We don't have live anymore so I wonder if we can get ride of it?

Reply:
done